index.js

```diff
@@ -105,7 +107,12 @@ function fastifyStatic (fastify, opts, next) {
 
   if (opts.decorateReply !== false) {
     fastify.decorateReply('sendFile', function (filePath) {
-      pumpSendToReply(this.request, this, filePath)
+      return new Promise((resolve, reject) => {
+        pumpSendToReply(this.request, this, filePath, (err) => {
+          if (err) reject(err)
+          else resolve()
+        })
+      })
```

Comment:
Can you add a catch handler, so that this does not become an unhandledRejection?

Reply:
Then we can directly only use `resolve` ?

Reply:
@mcollina WDYT?